winui/Masked-TextBox/How-To.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: Validation support in WinUI MaskedTextBox control | Syncfusion
+description: Learn all about the validation support in the MaskedTextBox (SfMaskedTextBox) control.
+platform: WinUI
+control: SfMaskedTextBox
+documentation: ug
+---
+
+# How to restrict the validation in WinUI MaskedTextBox
+
```

Comment:
@BhaskarSuresh22 How to restrict the default validation in WinUI MaskedTextBox

Reply:
@Manivannan-E, Done the changes.